UPGRADE.md

```diff
@@ -2,6 +2,16 @@
 
 ## dev-develop
 
+### Upgrade js dependencies
+
+**This change only affects you if you have used a 2.0.0 alpha release before**
+
+The following js dependencies in your `package.json` need to be updated:
+
+```bash
+npm install --save-dev raw-loader@^2.0 clean-webpack-plugin@^2.0 css-loader@^1.0 file-loader@^3.0 whatwg-fetch@^3.0 url-search-params-polyfill@^6.0 mini-css-extract-plugin@^0.6 autoprefixer@^9.5 postcss-calc@^7.0 postcss-import@^12.0 postcss-nested@^4.0 postcss-simple-vars@^5.0
```

Comment:
Until now we've said that `sulu-minimal` should be checked to see if something changed... I am slowly worried that we blow up our `UPGRADE.md` in a way that nobody reads it anymore :confused:

Reply:
In my opinion all changes which are required should be added to the UPGRADE.md. In the skeleton there are just sometimes changes which are optional which are not required.